node/network/gossip-support/src/lib.rs

```diff
@@ -109,7 +109,7 @@ fn choose_random_subset<T>(mut v: Vec<T>) -> Vec<T> {
 	v.shuffle(&mut rng);
 
 	let sqrt = (v.len() as f64).sqrt() as usize;
-	let len = std::cmp::max(25, sqrt);
+	let len = std::cmp::max(4, sqrt);
```

Comment:
the comment above is outdated
> at least 25 elements

Reply:
uh right, should not matter though as this change should get reverted as soon as we finished testing.